packages/main/src/SliderBase.ts

```diff
@@ -719,11 +717,19 @@ abstract class SliderBase extends UI5Element {
 		return isUpAction ? step : step * -1;
 	}
 
-	static _isDecreaseValueAction(e: KeyboardEvent) {
+	static _isDecreaseValueAction(e: KeyboardEvent, directionStart: DirectionStart) {
```

Comment:
I was not able to find this function to be called anywhere in the project. I've tested in the test page to put breakpoint in it and aslo it's not called. Do we still need it?